numbat/src/lib.rs

```diff
@@ -190,6 +193,74 @@ impl Context {
         self.print_sorted(units, FormatType::Unit)
     }
 
+    pub fn print_help_for_keyword(&mut self, keyword: &str) -> Markup {
+        let reg = self.interpreter.get_unit_registry();
+
+        if let PrefixParserResult::UnitIdentifier(_span, prefix, _, full_name) =
+            self.prefix_transformer.prefix_parser.parse(keyword)
```

Comment:
I'm not sure if using the `prefix_parser` is the best way to do this (since it required me making it `pub`)

Reply:
I think it's fine